src/main/java/org/gridsuite/study/server/service/FrontService.java

```diff
@@ -0,0 +1,8 @@
+package org.gridsuite.study.server.service;
+
+/**
+ * An enum that list know front apps
+ */
+public enum FrontService {
```

Comment:
Is a web app a service ?

Reply:
yes (for now)

Reply:
I'm not sure, usually services is for back-end, but whatever